R/matlab.R

```diff
@@ -0,0 +1,59 @@
+#' Run a matlab process
+#'
+#' This function returns a list of functions to start/run/stop a matlab process.
+#' The code is sent to matlab via a socket connection, and the results are
+#' written back in another socket connection.
+#' @param port A TCP port number
+#' @return A list of functions \code{start()}, \code{exec()}, \code{running()}
+#'   (check if the process has been running), and \code{stop()}.
+#' @export
+#' @examples \dontrun{
+#' mat = proc_matlab()
+#' mat$start()
+#' mat$exec('1+1')
+#' mat$exec('x = 1 + 1; x = x + x; x;') # return nothing
+#' mat$exec('5:9') # [     5     6     7     8     9]
+#' mat$running() # should be TRUE
+#' mat$stop()
+#' }
+
+proc_matlab <- function(port = 6011, existing=FALSE){
+  require(R.matlab)
```

Comment:
Please avoid `require()`. Import functions from the package instead, or just use `R.matlab::function()`. The latter seems to be more appropriate in this case.